app/src/main/java/org/schabi/newpipe/player/notification/NotificationUtil.java

```diff
@@ -167,19 +167,15 @@ public boolean shouldUpdateBufferingSlot() {
                 && notificationBuilder.mActions.get(2).actionIntent != null);
     }
 
-
+    @SuppressLint("NewApi") // ServiceInfo constant is inlined
```

Comment:
Are you sure this is the case? Can you provide a source? We can just keep using `if {} else {}`, it's not too bad after all

Reply:
https://stackoverflow.com/questions/12321804/android-version-checking-and-inlining-of-static-final-constants

Also, a lot of the library classes such as NotificationManagerCompat use the same trick.

Reply:
Mmmh, I would still stay on the safe side and use `if {} else {}`